docs/source/daml-script/index.rst

```diff
@@ -297,23 +297,32 @@ translated to Daml script but there are a few things to keep in mind:
 
 .. _daml-script-distributed:
 
-Use Daml Script in Distributed Topologies
-=========================================
+Use Daml Script in Canton
+=========================
 
 So far, we have run Daml script against a single participant node. It
 is also more possible to run it in a setting where different parties
 are hosted on different participant nodes. To do so, pass the
-``--participant-config participants.json`` file to ``daml script``
-instead of ``--ledger-host`` and ``ledger-port``. The file should be of the format
+``--participant-config participant-config.json`` file to ``daml script``
+instead of ``--ledger-host`` and ``ledger-port``.
+You can generate this file by calling [`utils.generate_daml_script_participants_conf(defaultParticipant = Some(one))`]
+(https://docs.daml.com/canton/usermanual/console.html#utils-generate-daml-script-participants-conf)
+in the canton console or in [the bootsrap scripts]
+(https://docs.daml.com/canton/tutorials/getting_started.html#automation-using-bootstrap-scripts)
 
-.. literalinclude:: ./participants-example.json
+The file should be of the format
+
+.. literalinclude:: ./participants-config.json
    :language: json
 
-This will define a participant called ``one``, a default participant
-and it defines that the party ``alice`` is on participant
+This will define a participant called ``one``, a default participant which refers
+to participant ``one`` and it defines that the party ``alice`` is on participant
 ``one``. Whenever you submit something as party, we will use the
 participant for that party or if none is specified
-``default_participant``. If ``default_participant`` is not specified,
+``default_participant``.
+
+If you use ``utils.generate_daml_script_participants_conf()``
+without a default particpant, the `default_participant` won't be defined and therefore
 using a party with an unspecified participant is an error.
 
 ``allocateParty`` will also use the ``default_participant``. If you
```

Comment:
I don't know if we can reference the `Setup` script in Daml App to give more examples of how to do so, ie `allocatePartyOn` ?

Reply:
We could but I probably wouldn’t bother for now.